roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -83,7 +83,7 @@ kubernetesMasterConfig:
 {% endif %}
   apiServerArguments: {{ api_server_args if api_server_args is defined else 'null' }}
   controllerArguments: {{ controller_args if controller_args is defined else 'null' }}
-  masterCount: {{ openshift.master.master_count }}
+  masterCount: {{ openshift.master.master_count if not openshift.master.cluster_method is defined or openshift.master.cluster_method == 'native' else 1 }}
```

Comment:
This might be a bit cleaner:
```
  masterCount: {{ openshift.master.master_count if openshift.master.cluster_method | default(None) == 'native' else 1 }}
```

I've started to yank the is defined tests in favor of using the deafult filter for tests like this.